notebooks/220-cross-lingual-books-alignment/220-cross-lingual-books-alignment.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<p>please add "" for requirements with versions and extras, not all platforms and environments can successfully interpret it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Fixed